packages/amplify-codegen/tests/commands/statements.test.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const { generate } = require('amplify-graphql-docs-generator');
+const { generate } = require('@aws-amplify/graphql-docs-generator');
```

Comment:
Why is this line get replaced? The similar part in the type generation test but is not replaced.

Reply:
I would suggest to have both package required and test it by switching the FF value to see if both `generate` gets called.

Reply:
I moved the legacy tests to a separate folder which can simply be deleted post deprecation of Feature flags. So these files now only test the API contracts between `amplify-codegen` and migrated packages.